### DISCUSSION TITLE
Welcome to gatsby-shopify-starter Discussions!

### DISCUSSION THREAD

**Discussion by AlexanderProd** [AlexanderProd/gatsby-shopify-starter#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heytulsiprasad** [comment]
Hey @AlexanderProd, thanks for making such a helpful project for everyone and it's nice to see more people use GitHub discussions. Really helps to ask small questions instead of bloating the issues.

I was just making a Shopify store and in that, I had to implement a **Buy Now** button in the product page which is just meant to send the user to the checkout page for only that specific product and quantity. 

So I felt if that'll be a good addition to this starter template, as most e-commerce sites have that functionality. I'd love to work on a PR if you like that feature to be on this template.

**Reply by AlexanderProd** [reply]
Hey @heytulsiprasad im glad you like my starter and it was of help for you. I do think so too that discussions is a great way to just ask quick questions. 

Im not sure what you mean by a Buy Now button and how you’d want to implement it in the starter, I mean there’s already a buy button for the products isn’t it?

**Reply by heytulsiprasad** [reply]
Yeah right, there's an *Add to Cart* button for now and people can then go to *Cart* and click on *Checkout* to checkout all their products at once, however there seems to be a **Buy now** button as well in sites for ex: Amazon where you just select the quantity/variant of one product and it checks that out (just that product) without adding to cart.

Snapshot:
![image](https://user-images.githubusercontent.com/52394293/102322084-85635c80-3fa4-11eb-9221-13d2b8cdc5ee.png)

Ideally, the buy now button would redirect to the shopify checkout page with the product one clicked buy for.

**Reply by AlexanderProd** [reply]
Oh I see, so it basically just skips the cart process
